server/public/index.html

```diff
@@ -2,6 +2,8 @@
   <head>
     <link rel="stylesheet" href="style.css">
     <script src='./bundle.js' defer></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://kit.fontawesome.com/648216d052.js" crossorigin="anonymous"></script>
```

Comment:
I don't know how heavy a font kit is, but also seems like we don't need a full one just for controls. Just add a couple of images to the proejct?

Reply:
keep font kit